dojo/importers/importer/importer.py

```diff
@@ -356,10 +309,34 @@ def import_scan(self, scan, scan_type, engagement, lead, environment, active, ve
             parsed_findings = parser.get_findings(scan, test)
 
         logger.debug('IMPORT_SCAN: Processing findings')
-        new_findings = self.process_parsed_findings(test, parsed_findings, scan_type, user, active,
-                                                    verified, minimum_severity=minimum_severity,
-                                                    endpoints_to_add=endpoints_to_add, push_to_jira=push_to_jira,
-                                                    group_by=group_by, now=now, service=service)
+        new_findings = []
+        if settings.ASYNC_FINDING_IMPORT:
+            chunk_list = importer_utils.chunk_list(parsed_findings)
+            results_list = []
+            # First kick off all the workers
+            for findings_list in chunk_list:
+                result = self.process_parsed_findings(test, findings_list, scan_type, user, active,
+                                                            verified, minimum_severity=minimum_severity,
+                                                            endpoints_to_add=endpoints_to_add, push_to_jira=push_to_jira,
+                                                            group_by=group_by, now=now, service=service, sync=False)
+                # Since I dont want to wait until the task is done right now, save the id
+                # So I can check on the task later
+                results_list += [result]
+            # After all tasks have been started, time to pull the results
+            logger.info('IMPORT_SCAN: Collecting Findings')
+            for results in results_list:
+                serial_new_findings = results.get()
+                new_findings += [next(serializers.deserialize("json", finding)).object for finding in serial_new_findings]
+            logger.info('IMPORT_SCAN: All Findings Collected')
+            # Indicate that the test is not complete yet as endpoints will still be rolling in.
+            test.percent_complete = 50
+            test.save()
+            importer_utils.update_test_progess(testß)
```

Comment:
`ß` looks like a typo

Reply:
I'm not sure how to even type that key... Thanks for catching that! Fixed.